mobile/src/employee-details/employee-details.tsx

```diff
@@ -72,12 +95,41 @@ export class EmployeeDetailsImpl extends Component<EmployeeDetailsProps & Employ
                                 {contacts}
                             </View>
                         </View>
+
+                        {
+                            (events !== undefined && events.length > 0) ? 
+                            <FlatList
+                                data={events}
+                                keyExtractor={this.keyExtractor}
+                                renderItem={this.renderItem} /> : null
+                        }
+
                     </View>
                     </ScrollView>
                 </View>
         );
     }
 
+    private keyExtractor = (item: CalendarEvent) => item.calendarEventId;
+
+    private renderItem = (itemInfo: ListRenderItemInfo<CalendarEvent>) => {
```

Comment:
Please put this code in a separate child component, this one gets big

Reply:
ok